lib/cli/cli.js

```diff
@@ -48,7 +48,9 @@ exports.main = (argv = process.argv.slice(2)) => {
     .fail((msg, err, yargs) => {
       debug(err);
       yargs.showHelp();
-      console.error(`\n${symbols.error} ${ansi.red('ERROR:')} ${msg}`);
+      console.error(
+        `\n${symbols.error} ${ansi.red('ERROR:')} ${err ? err.message : msg}`
+      );
```

Comment:
What is the reason for this change?

Reply:
Not sure anymore. Probably an artifact of my earlier playing around with async/sync dual code. The fact that everything is now async made it simpler. 

Removed the diff. It's now back to to what it was.